.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
To be honest I'm more of a fan of direct options? Why not making this part of logger hash? Seems a bit excessive having separate file, no?

Reply:
This file is not for logging or script settings, it is for the editor in which the script is modified.
Usually I use tabs for indents, this is the default value of my editor, but in this project spaces are used. This is the reason for the "fix indent" commits.
If my editor sees such a file, he redefines his values for this project, and I, or someone else, no longer worries about indents.
https://editorconfig.org/